lib/manageiq/cross_repo/repository.rb

```diff
@@ -45,6 +45,21 @@ def parse_identifier(identifier, server)
         repo = path.basename.to_s
         ref  = Dir.chdir(path) { `git rev-parse HEAD`.chomp }
         sha  = ref
+      elsif identifier.start_with?(server)
+        identifier = URI.parse(identifier).path
+        org_and_repo, pr = identifier.split("/pull/")
+        _, org, repo = org_and_repo.split("/")
```

Comment:
The path has a leading `/` e.g. `/ManageIQ/manageiq/pull/1234` so I could either strip it out then split, or just split and ignore the first component

Reply:
This is fine.